### ISSUE TITLE
Consequences of a file with complicated merges

### ISSUE THREAD

**Issue by baumannsven** [phpcq/author-validation#13]
So we have a problem for detect the author for files, their create in deep merges. This files can we actual not follow.

Here is a sample, where the follow dismissed. 

https://github.com/contao-community-alliance/dc-general/blob/a692679aae9e04a89fa06be472e7fa300b028a5b/src/Contao/View/Contao2BackendView/EventListener/SelectModeButtonsListener.php

**Comment by baumannsven** [comment]
I have tested somethings for this problem. I have a idea, for follow them.
Here is a example where i follow:

```bash
git diff a692679aae9e04a89fa06be472e7fa300b028a5b e08c9385dee352864ff4061265571776d5cf8fe7 --diff-filter=R --name-status -- src/Contao/View/Contao2BackendView/EventListener/SelectModeButtonsListener.php src/ContaoCommunityAlliance/DcGeneral/Contao/View/Contao2BackendView/EventListener/SelectModeButtonsListener.php
```

The test of this sounds good. Tested of another file is good. I hope is the right way.

**Comment by baumannsven** [comment]
Fixed in 569ef61588817e44e18f92d7ba44165e26cb8fd2
